torchgeo/datasets/bigearthnet.py

```diff
@@ -143,50 +143,50 @@ class BigEarthNet(NonGeoDataset):
             "Marine waters",
```

Comment:
Can you add a comment pointing to https://git.tu-berlin.de/rsim/BigEarthNet-S2_19-classes_models/-/blob/master/label_indices.json at the top of all of this so we know where the definitive source is?